src/main/java/learn/reservations/ui/View.java

```diff
@@ -0,0 +1,106 @@
+package learn.reservations.ui;
+
+import learn.reservations.data.DataAccessException;
+import learn.reservations.models.Reservation;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Scanner;
+
+
+@Component
+public class View {
+
+    private final Scanner console = new Scanner(System.in);
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy");
+
+    public MainMenuOption selectMenuOption() {
+        System.out.println("Main Menu");
+        System.out.println("=========");
+        for (MainMenuOption option : MainMenuOption.values()){
+            System.out.printf("%s. %s%n", option.getValue(), option.getMessage());
+        }
+        System.out.println("Select [0-4]");
+        int numberOption = Integer.parseInt(console.nextLine());
```

Comment:
This crashes the app unless I provide an int, a `readInt` method that handles that error gracefully would be better.